networking/gcp/vpc/versions.tf

```
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.0"
```

```suggestion
      version = ">= 4.0"
```

I think the version 3.0 is a bit old.